packages/block-library/src/image/edit.native.js

```diff
@@ -288,11 +278,11 @@ class ImageEdit extends React.Component {
 					{ ( ! this.state.isCaptionSelected ) &&
 						getToolbarEditButton( openMediaOptions )
 					}
-					<InspectorControls>
+					<InspectorControls buttons>
 						<ToolbarButton
 							title={ __( 'Image Settings' ) }
 							icon="admin-generic"
-							onClick={ onImageSettingsButtonPressed }
+							onClick={ onSettingsToggle }
```

Comment:
pass handler from the 'block.native.js` to open/close `BottomSheet` with options

Reply:
Ok I see your point. The settings button should be common and available for all blocks (sometimes it won't be visible because block do not have any settings) and I agree it make sense to move it to `BlockMobileToolbar`.

I will also check what's the option for pass prop to conditionally show/hide it if block has settings

Reply:
condition for hide/show settings is finished